cmd/apiserver/app/server/run_server.go

```diff
@@ -33,9 +33,8 @@ import (
 // RunServer runs an API server with configuration according to opts
 func RunServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error {
 	if stopCh == nil {
-		/* the caller of RunServer should generate the stop channel
-		if there is a need to stop the API server */
-		stopCh = make(chan struct{})
+		/* whe RespectsStopCh: true and the stop channel is nil, error returned*/
```

Comment:
I'm not really sure what this comment is trying to say. "Return an error if stopCh is nil"? If that's what it's trying to say, maybe it should be on like 34, above the branch?